### PR TITLE
Completed test case CG15

### PR DESCRIPTION
**Pull request by MattMiss**
# Description

Added CG15 test case in readingsCompareGroupFlow.js

Partly Addresses #962 

## Type of change

- [ ] Note merging this changes the database configuration.
- [ ] This change requires a documentation update

## Checklist

- [x] I have followed the [OED pull request](https://openenergydashboard.github.io/developer/pr.html) ideas
- [x] I have removed text in ( ) from the issue request
- [x] You acknowledge that every person contributing to this work has signed the [OED Contributing License Agreement](https://openenergydashboard.github.io/developer/cla.html) and each author is listed in the Description section.

## Limitations

None

### CODE REVIEW

File: src/server/test/web/readingsCompareGroupFlow.js [hunk 1]
```diff
@@ -7,19 +7,116 @@
...
+	conversionDatakWh
+ } = require('../../util/readingsUtils');
 
 mocha.describe('readings API', () => {
 	mocha.describe('readings test, test if data returned by API is as expected', () => {
 		mocha.describe('for compare charts', () => {
 			mocha.describe('for groups', () => {
-				// Add CG15 here
+				// Test CG15					
+				mocha.it(' 7 day shift end 2022-10-31 17:00:00 for 15 minute reading intervals and flow units & kW as kW ', async () => {
```
Comment (huss) [inline src/server/test/web/readingsCompareGroupFlow.js]: Could you please add CG15: at the start of the description?

**Review by huss — CHANGES_REQUESTED** [review]
Thanks to @MattMiss for their first contribution to OED. Review and testing found it works as expected. I made one small comment on a string to address.

File: src/server/test/web/readingsCompareGroupFlow.js (hunk 1 above)
Comment (MattMiss) [reply src/server/test/web/readingsCompareGroupFlow.js]: I have added CG15 to the test description

**Review by huss — APPROVED** [review]
Thanks to @MattMiss for the updated code. Everything looks good. Congratulations on your first accepted contribution to OED.

**Pull request merged**